### ISSUE TITLE
Derive labels from PR title

### ISSUE THREAD

**Issue by asos-danieltavares** [davdroman/Badonde#59]
Would be nice to derive labels from PR title.

Example. [TICKETNUMBER] My Lovely PR fixes some UI Tests.

Should try and include UI Tests label

**Comment by davdroman** [comment]
De-scoped from core functionality, this will be possible to implement on a team-per-team basis after #60.
